README.md

```diff
@@ -0,0 +1,171 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>🚗Test Drive Zone🚙</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [🚗Test Drive Zone🚙 ](#test-drive-zone-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [Link to Frontend App](#link-to-frontend-app)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 🚗Test Drive Zone🚙 <a name="about-project"></a>
+
+This app allows users to schedule test drive appointments for cars. Users can add, delete, and view details of available cars, making it easy to manage appointments for test drives.
+
+## 🛠 Built With <a name="built-with"></a>
+Ruby on Rails
+
+### Tech Stack <a name="tech-stack"></a>
+
+- <a href="https://www.postgresql.org/">PostgreSQL</a>
+- <a href="https://rubyonrails.org/">Ruby on Rails</a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+Key features of the application
+
+- Add Cars : Users can add cars to the inventory
+- Delete Cars: Users can remove car as needed.
+- Reserve Cars : Schedule test drive appointments for specific cars.
+- View Car Details: Explore detailed information about each car available for test drives.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Frontend Link -->
+
+## Link to Frontend App
+
+- [Front-end](https://github.com/SanaTameem/book-an-appointment-front-end.git)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need
+- laptop
+- git
+- Code Editor.
+- PostgreSQL.
+- Ruby On Rails.
+- Linters
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/SanaTameem/book-an-appointment-back-end.git
+```
+
+### Install
+
+Install this project with:
+
+- Install <a href="https://www.postgresql.org/">PostgreSQL</a>.
+- Install <a href="https://rubyonrails.org/">Ruby On Rails</a> using `rails new book-an-appointment --api --database=postgresql` command.
+```sh
+  cd my-project
+  bundle install
+```
```

Comment:
- [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate, rails db:seed``. 👍